hdf/test/testhdf.c

```diff
@@ -268,8 +268,12 @@ main(int argc, char *argv[])
 
     if (CleanUp) {
         MESSAGE(2, printf("\nCleaning Up...\n\n"););
+#if !(defined DOS386 || defined WIN386)
```

Comment:
These aren't defined anymore, so you want:

`#ifndef H5_HAVE_WIN32_API`

as the test.